src/config.ts

```diff
@@ -9,13 +9,19 @@ import { Provider } from 'web3/providers';
 
 const CONFIG_FILE = 'test-environment.config.js';
 
-type Config = {
+export type Config = {
   accounts: { amount: number; ether: number };
   contracts: { type: string; defaultGas: number; defaultGasPrice: number; artifactsDir: string };
   blockGasLimit: number;
```

Comment:
This one looks like it should go in `node` as well. But it should be called `gasLimit` because that's what Ganache expects.

Reply:
Yeah, but it is already there and that would mean breaking backward compatibility. I know we are at 0.x version but still. Let's how it goes.

Reply:
We should support it for a while with a deprecation message, while we allow people to migrate to the new config format.